src/languages/en.js

```diff
@@ -846,4 +846,13 @@ export default {
         screenShare: 'Screen share',
         screenShareRequest: 'Expensify is inviting you to a screen share',
     },
+    genericErrorPage: {
+        title: 'Uh-oh, something went wrong!',
+        body: {
+            helpTextMobile: 'Please try closing and reopening the app or switching to ',
+            helpTextWeb: 'web.',
```

Comment:
IIRC punctuation should not be included in translations.

```suggestion
            helpTextWeb: 'web',
```

Right now we have translations that include exclamation points like `Peanuts!` and `¡Cacahuates!`. It's out of scope for this PR, but we should probably create utility functions like:

```javascript
// In src/libs/Localize.js
function exclaim(locale, str) {
    switch (locale) {
        case 'es':
        case 'es-ES':
            return `¡${str}!`;
        case 'en':
        default:
            return `${str}!`;
    }
}
```

Reply:
Should I add a utility function here? then?

Reply:
> IIRC punctuation should not be included in translations.

@roryabraham  I checked all the previous translations and punctuations are all a part of the translation only.   Did we change this recently? If yes should we create a separate issue to update all the translations?

Reply:
Sorry, my mistake. Nothing more to do here. We can leave this for now to be consistent with our other translations.